test/tests.js

```diff
@@ -1,7 +1,5 @@
-/* eslint-disable no-undef */
-
-describe('Appboy Forwarder', function () {
-    var expandCommerceEvent = function (event) {
+describe('Appboy Forwarder', function() {
+    var expandCommerceEvent = function(event) {
```

Comment:
```suggestion
    const expandCommerceEvent = function(event) {
```

Reply:
not sure why this was even changed, i'm reverting the change and avoiding any var--> const changes here. When I make the change I get a linting issue. Could we agree that as a general rule lets keep those changes to TS given that eslint may not agree and throw linting errors?